packages/panzer/disc-fe/src/evaluators/Panzer_Integrator_CurlBasisDotVector.hpp

```diff
@@ -40,92 +40,339 @@
 // ***********************************************************************
 // @HEADER
 
-#ifndef PANZER_EVALUATOR_CURLBASISDOTVECTOR_DECL_HPP
-#define PANZER_EVALUATOR_CURLBASISDOTVECTOR_DECL_HPP
+#ifndef   __Panzer_Integrator_CurlBasisDotVector_hpp__
+#define   __Panzer_Integrator_CurlBasisDotVector_hpp__
```

Comment:
Why was this changed? The leading double underscore is usually reserved for __ compiler defines. I have tried to avoid this. I know it is not consistent throughout panzer...

Reply:
An artifact of however I learned C++&mdash;"the `#define` should be the file name, with an underscore in place of the period, surrounded by double underscores"&mdash;not sure where that guidance came from though.  Hadn't heard about that being reserved for compiler defines.